### ISSUE TITLE
Create MySQL database

### ISSUE THREAD

**Issue by IceFlovver** [amosproj/amos2022ws01-firmware-scraper#15]
## User story
1. As a user
2. I need a MySQL database to be set up
3. So that entries can be later added to it

## Acceptance criteria
* A MySQL database exists

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
